src/main/java/com/softserveinc/ita/pageobjects/ContactPage.java

```diff
@@ -0,0 +1,94 @@
+package com.softserveinc.ita.pageobjects;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+
+import java.util.List;
+
+import static com.codeborne.selenide.CollectionCondition.*;
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$$x;
+import static com.codeborne.selenide.Selenide.$x;
+import static java.time.Duration.ofSeconds;
+import static java.util.stream.Collectors.*;
+
+public class ContactPage extends BasePage<ContactPage> {
+
+    private final SelenideElement searchSelector = $x("//div[@Class='autocomplete']/input[@name='search']");
+    private final ElementsCollection listOfCitiesSelector = $$x("//ul[@class='autocomplete__list dialog']/li");
+
+    public ContactPage fillCitySearchField(String expectedCity) {
+        searchSelector
+                .shouldBe(visible)
+                .click();
+        searchSelector.clear();
+        searchSelector.setValue(expectedCity);
+
+        return this;
+    }
+
+    public String getErrorMessage() {
+        return $x("//div[@class='autocomplete']//p")
+                .getText()
+                .trim();
+    }
+
+    public String getActualCity(String expectedCity) {
+        return listOfCitiesSelector
```

Comment:
it possible only if this list is visible,

Reply:
why do you need to get the same city, what you pass through agrs? probably better to get list in test and verify if this list contains expected city

Reply:
i`ve tried to collect list and verify if it contains actual city, but there is whitespaces and my test was failed many times because of it